src/Func.h

```diff
@@ -853,6 +853,12 @@ class Func {
      * splitting it up. If bounds inference decides that it requires
      * more of this function than the bounds you have stated, a
      * runtime error will occur when you try to run your pipeline.
+     *
+     * The min argument may be an undefined Expr, in which case the
```

Comment:
Func::bound_extent(Expr)?

On Tue, Apr 12, 2016 at 3:26 PM, Dillon Sharlet <notifications@github.com>
wrote:

> In src/Func.h
> <https://github.com/halide/Halide/pull/1117#discussion_r59464962>:
>
> > @@ -853,6 +853,12 @@ class Func {
> >       * splitting it up. If bounds inference decides that it requires
> >       * more of this function than the bounds you have stated, a
> >       * runtime error will occur when you try to run your pipeline.
> > +     *
> > +     * The min argument may be an undefined Expr, in which case the
>
> I can't think of exactly what it should be, but it seems like there should
> be a bound overload or equivalent that only takes an extent. Using
> undefined Exprs is only surfaced to users in a few places, and it's easy to
> confuse with undef().
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/halide/Halide/pull/1117/files/ee0190f4fdde616512eb0d12d7433bafe6ae8bf9#r59464962>
>